Changes to example page html for listbox-combo.html and LayoutGrid.html

spectranaut opened the pull request:
Hi @mcking65 -- per our discussion, here is a PR to master with the changes to the example pages themselves. This commit only add "data-test-id" to the appropriate tables.

mcking65 commented on examples/combobox/aria1.1pattern/listbox-combo.html:
```diff
@@ -274,7 +274,7 @@ <h3 id="kbd_label_listbox">Listbox Popup</h3>
             </td>
           </tr>
-          <tr>
+          <tr data-test-id="key-down-arrow">
```
This is same ID as line 185, but won't the tests be different?

spectranaut replied:
you are right, and it lead to me to not test this row of this table. Changing this to: `listbox-key-down-arrow` and the other to `textbox-key-down-arrow`

mcking65 commented on examples/combobox/aria1.1pattern/listbox-combo.html:
```diff
@@ -264,7 +264,7 @@ <h3 id="kbd_label_listbox">Listbox Popup</h3>
             </td>
           </tr>
-          <tr>
+          <tr data-test-id="key-escape">
```
This is same ID as line 222, but won't tests be different?

spectranaut replied:
Changing this to: `listbox-key-escape` and the other to `textbox-key-escape`

mcking65 commented on examples/combobox/aria1.1pattern/listbox-combo.html:
```diff
@@ -284,7 +284,7 @@ <h3 id="kbd_label_listbox">Listbox Popup</h3>
             </td>
           </tr>
-          <tr>
+          <tr data-test-id="key-up-arrow">
```
This is same ID as line 199, but won't tests be different?

spectranaut replied:
Changing this to: `listbox-key-up-arrow` and the other to `textbox-key-up-arrow`

spectranaut commented:
Hey @mcking65 addressed your feedback. I'll have to add a few more tests to listbox-combo.js -- so good catch. I'll add that to the backlog issue #780

mcking65 commented:
looks good
